src/App.tsx

```diff
@@ -32,172 +87,87 @@ export const App: React.FC = () => {
-              ×
-            </button>
-
-            {/* 'is-active' class puts this modal on top of the todo */}
-            <div data-cy="TodoLoader" className="modal overlay is-active">
-              <div className="modal-background has-background-white-ter" />
-              <div className="loader" />
-            </div>
-          </div>
+          {isLoading ? <Loader /> : <TodoList todos={filteredTodos} />}
         </section>
-
-        {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
-
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
-
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
-            >
-              Completed
-            </a>
-          </nav>
-
-          {/* this button should be disabled if there are no completed todos */}
-          <button
-            type="button"
-            className="todoapp__clear-completed"
-            data-cy="ClearCompletedButton"
-          >
-            Clear completed
-          </button>
-        </footer>
+        {todos.length > 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {todos.filter(todo => !todo.completed).length} items left
+            </span>
+            <nav className="filter" data-cy="Filter">
+              <nav className="filter" data-cy="Filter">
+                <a
+                  href="#/"
+                  className={`filter__link ${filterStatus === 'all' ? 'selected' : ''}`}
+                  data-cy="FilterLinkAll"
+                  onClick={() => setFilterStatus('all')}
+                >
+                  All
+                </a>
+                <a
+                  href="#/active"
+                  className={`filter__link ${filterStatus === 'active' ? 'selected' : ''}`}
+                  data-cy="FilterLinkActive"
+                  onClick={() => setFilterStatus('active')}
+                >
+                  Active
+                </a>
+                <a
+                  href="#/completed"
+                  className={`filter__link ${filterStatus === 'completed' ? 'selected' : ''}`}
+                  data-cy="FilterLinkCompleted"
+                  onClick={() => setFilterStatus('completed')}
+                >
+                  Completed
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method